tests/unit/x509.js

```diff
@@ -406,6 +471,49 @@ var UTIL = require('../../lib/util');
       });
     });
 
+    it('should generate a self-signed certificate after 2050', function() {
+      var keys = {
+        privateKey: PKI.privateKeyFromPem(_pem.privateKey),
+        publicKey: PKI.publicKeyFromPem(_pem.publicKey)
+      };
+	  var attrs = [{
+        name: 'commonName',
+        value: 'example.org'
+      }, {
+        name: 'countryName',
+        value: 'US'
+      }, {
+        shortName: 'ST',
+        value: 'Virginia'
+      }, {
+        name: 'localityName',
+        value: 'Blacksburg'
+      }, {
+        name: 'organizationName',
+        value: 'Test'
+      }, {
+        shortName: 'OU',
+        value: 'Test'
+      }];
+      let notBefore = new Date("2050-02-02");
```

Comment:
Use 'var' to fix the test suite.  We'll eventually move everything to ES6+ in one huge change.